frontend/common/src/components/accordion/Accordion.tsx

```diff
@@ -38,10 +38,10 @@ const Accordion: React.FC<AccordionProps> = ({
     >
       <button
         type="button"
-        data-h2-text-align="b(left)"
```

Comment:
Had to pull out a lot of this styling out of h2 so that the printing can use the styling, too.